pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -1,30 +1,33 @@
 { stdenv, fetchurl, fetchgit, buildPythonPackage, python, pythonOlder,
   cudaSupport ? false, cudatoolkit ? null, cudnn ? null, nccl ? null, magma ? null,
-  mklSupport ? false, mkl ? null,
+  mklSupport ? false, mkl ? null, openblas ? null,
   openMPISupport ? false, openmpi ? null,
-  buildNamedTensor ? false,
   buildBinaries ? false,
+  buildDocs ? false,
+  useNixProtobuf ? false,
   cudaArchList ? null,
-  fetchFromGitHub, lib, numpy, pyyaml, cffi, click, typing, cmake, hypothesis, numactl,
+  fetchFromGitHub, lib, numpy, pyyaml, cffi, click, typing, cmake, hypothesis, numactl, psutil,
   linkFarm, symlinkJoin,
 
   # ninja (https://ninja-build.org) must be available to run C++ extensions tests,
   ninja,
 
   # dependencies for torch.utils.tensorboard
-  tensorboardSupport ? true, pillow, six, future, tensorflow-tensorboard,
+  pillow, six, future, tensorflow-tensorboard, protobuf,
 
   utillinux, which, isPy3k }:
 
 assert !openMPISupport || openmpi != null;
-assert !tensorboardSupport || tensorflow-tensorboard != null;
 
 # assert that everything needed for cuda is present and that the correct cuda versions are used
 assert !cudaSupport || cudatoolkit != null;
 assert cudnn == null || cudatoolkit != null;
 assert !cudaSupport || (let majorIs = lib.versions.major cudatoolkit.version;
                         in majorIs == "9" || majorIs == "10");
 
+# Using nix's protobuf instead of the bundled version with pytorch is considered broken.
+assert useNixProtobuf == false;
```

Comment:
then why allow the option if it's always wrong?

Reply:
Yeah... this was just hopeful code. I should have cleaned this up. Updating now.